nome.py

```diff
@@ -0,0 +1,30 @@
+saldo = 0
+
+while True:
+
+    print("Escolha uma dos números abaixo:")
+    print("1 - Ganho")
+    print("2 - Gasto")
+    print("3 - Saldo")
+    print("4 - Sair")
+
+    opcaoescolhida = input("Opção escolhida: ")
+
+
+    if opcaoescolhida == "1":
+        valor = float(input("Digite seu ganho: "))
+        saldo += valor
+        msg = (f"Ganho registrado! Agora você tem R${saldo:.2f}")
```

Comment:
Ao invés de chamar isso aqui, é melhor encapsular dentro de uma função.
```suggestion
        register_gain(ammount)
```